lib/cli.js

```diff
@@ -72,7 +72,21 @@ exports.run = () => {
 			'update-snapshots',
 			'color'
 		],
-		default: Object.assign({color: true}, conf),
+		default: {
+			init: conf.init,
+			failFast: conf.failFast,
+			serial: conf.serial,
+			cache: conf.cache,
+			powerAssert: conf.powerAssert,
+			tap: conf.tap,
+			verbose: conf.verbose,
+			watch: conf.watch,
+			match: conf.match,
+			timeout: conf.timeout,
+			concurrency: conf.concurrency,
+			updateSnapshots: conf.updateSnapshots,
+			color: true || conf.color
```

Comment:
I think this should be `'color' in conf ? color.conf : true`. The idea is that it defaults to `true` if not configured at all, but otherwise the config value should prevail.

It's strange that this doesn't fail any tests though.